src/Forms/Shared/Samples/Layers/EditKmlGroundOverlay/EditKmlGroundOverlay.xaml

```diff
@@ -0,0 +1,31 @@
+﻿<ContentPage
+    x:Class="ArcGISRuntimeXamarin.Samples.EditKmlGroundOverlay.EditKmlGroundOverlay"
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+        <esriUI:SceneView x:Name="MySceneView" />
```

Comment:
I recommend adding an explicit `Grid.Row="0"` here.

Additionally, I recommend explicitly setting a height for the row definition above.